### PR TITLE
Annotations Parameter - Description bugfix

### PR DESCRIPTION
**Pull request by jesscall**
Bug : 

```
Traceback (most recent call last):
  File "/opt/eegnet/bin/mri/python/bids_import.py", line 464, in <module>
    main()
  File "/opt/eegnet/bin/mri/python/bids_import.py", line 76, in main
    read_and_insert_bids(bids_dir, config_file, verbose, createcand, createvisit)
  File "/opt/eegnet/bin/mri/python/bids_import.py", line 230, in read_and_insert_bids
    loris_bids_root_dir    = loris_bids_root_dir
  File "/opt/eegnet/bin/mri/python/lib/eeg.py", line 155, in __init__
    self.register_data()
  File "/opt/eegnet/bin/mri/python/lib/eeg.py", line 284, in register_data
    derivatives
  File "/opt/eegnet/bin/mri/python/lib/eeg.py", line 757, in fetch_and_insert_annotation_files
    annotation_metadata, annotation_metadata_path, physiological_file_id, blake2
  File "/opt/eegnet/bin/mri/python/lib/physiological.py", line 612, in insert_annotation_metadata
    annotation_metadata['Author']
  File "/opt/eegnet/bin/mri/python/lib/database_lib/physiologicalannotationparameter.py", line 40, in insert
    values       = (annotation_file_id, sources, author)
  File "/opt/eegnet/bin/mri/python/lib/database.py", line 190, in insert
    raise Exception("Insert query failure: " + format(err))
Exception: Insert query failure: (1364, "Field 'Description' doesn't have a default value")
```

The description field is added in [this PR](https://github.com/aces/Loris/pull/7433/files#diff-b669a1cbe43867823d7c1fe00fa1a00f03a478b1caa8fa150345cf11672946a6R268)

### CODE REVIEW

File: python/lib/database_lib/physiologicalannotationparameter.py [hunk 1]
```diff
@@ -20,13 +20,16 @@ def __init__(self, db, verbose):
...
 
-    def insert(self, annotation_file_id, sources, author):
+    def insert(self, annotation_file_id, description, sources, author):
         """
         Inserts a new entry in the physiological_annotation_parameter table.
 
         :param annotation_file_id : annotation file's ID
          :type annotation_file_id : int
 
+        :param description        : Description of the derivative data
```
Comment (jesscall) [inline python/lib/database_lib/physiologicalannotationparameter.py:30]:
```suggestion
        :param description        : Description of the annotations
```

**Comment by cmadjar** [comment]
@jesscall I checked out your branch and it looks like some of the bug fixes that were fixed on main are back on your branch but don't show in the diff. Could you rebase your branch on main?

**Comment by jesscall** [comment]
@cmadjar done!

**Review by cmadjar — APPROVED** [review]
LGTM

**Pull request merged**